src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/DataHistoryWidgetBuilder.tsx

```diff
@@ -132,7 +134,8 @@ const DataHistoryWidgetBuilder: React.FC<IDataHistoryWidgetBuilderProps> = ({
     }, [adxConnectionInformation]);
 
     const connectionString = formData.widgetConfiguration.connection
-        ? generateConnectionString(formData.widgetConfiguration.connection)
+        ? generateConnectionString(formData.widgetConfiguration.connection) ||
+          t(LOC_KEYS.connectionStringInvalid)
```

Comment:
if connection string generation fails due to invalid adx url, shows invalid url message in the input field